proto/frequenz/api/dispatch/dispatch.proto

```diff
@@ -116,31 +116,26 @@ message DispatchComponentIDs {
   repeated uint64 component_ids = 1;
 }
 
-// Parameters for filtering the dispatch list
-message DispatchFilter {
-  // Filter by dispatch ID
-  repeated uint64 ids = 1;
-
-  // Filter by microgrid ID
-  repeated uint64 microgrid_ids = 2;
+// Message for listing dispatches for a given microgrid, and an optional filter
+message DispatchListRequest {
+  // The microgrid ID
+  uint64 microgrid_id = 1;
 
-  // Filter by dispatch type
-  repeated string types = 3;
+  // Additional filter parameters
+  DispatchFilter filter = 2;
+}
 
+// Parameters for filtering the dispatch list
+message DispatchFilter {
   // Filter by component ID or category
-  repeated DispatchComponentSelector selectors = 4;
+  repeated DispatchComponentSelector selectors = 1;
 
   // Filter by time interval
-  TimeIntervalFilter time_interval = 5;
-
-  // Filter by "active" status
-  optional bool is_active = 6;
```

Comment:
Do I understand correctly that now you can't get only active dispatched for example?